SQL/Archive/17.1/2017-07-13-session-default-null.sql

```diff
@@ -0,0 +1 @@
+alter table session MODIFY COLUMN MRIQCStatus enum('','Pass','Fail') NULL DEFAULT NULL;
```

Comment:
These schema changes should no longer be required, now that the code uses `Utility::nullifyEmpty`